src/pages/en/guides/migrate-to-astro/from-nextjs.mdx

```diff
@@ -2,72 +2,924 @@
+You can also pass a `--template` argument to the `create astro` command to start from an existing Astro repository on GitHub:
 
   <PackageManagerTabs>
     <Fragment slot="npm">
     ```shell
-    npm create astro -- --template withastro/blog
+    # launch the Astro CLI Wizard 
+    npm create astro@latest
+
+    # start from any existing repo
+    npm create astro -- --template <github-username>/<github-repo>
     ```
     </Fragment>
     <Fragment slot="pnpm">
     ```shell
-    pnpm create astro --template withastro/blog
+    # launch the Astro CLI Wizard 
+    pnpm create astro@latest
+
+    # start from any existing repo
+    pnpm create astro --template <github-username>/<github-repo>
     ```
     </Fragment>
     <Fragment slot="yarn">
     ```shell
-    yarn create astro --template withastro/blog
+    # launch the Astro CLI Wizard 
+    yarn create astro@latest
+
+    # start from any existing repo
+    yarn create astro --template <github-username>/<github-repo>
     ```
     </Fragment>
   </PackageManagerTabs>
 
-Bring your existing Markdown (or MDX, with our optional integration) files as content to [create Markdown or MDX pages](/en/guides/markdown-content/).
+Then, copy your existing Next project files over to your new Astro project in a separate folder outside of `src`.
+
+:::tip
+Prefer to explore in your browser instead? Visit https://astro.new for a variety of starter templates that can be opened and edited in StackBlitz, CodeSandbox, or Gitpod.
+:::
+
+### Install integrations (optional)
+
+You may find it useful to install some of [Astro's optional integrations](/en/guides/integrations-guide/) to use while converting your Next project to Astro:
+
+- **@astrojs/react**: to reuse some existing React UI components in your new Astro site, or keep writing with React components.
+
+- **@astrojs/image**: to replace Next's Image plugin with Astro's own image-optimizing components. (Experimental: works in `.astro` and `.mdx` files only.)
+
+- **@astrojs/mdx**: to bring existing MDX files from your Next project, or to use MDX in your new Astro site.
+
+### Put your source code in `src`
+
+1. **Keep** Next's `public/` folder untouched. 
+   
+    Astro uses the `public/` directory for static assets, just like Next. There is no change needed to this folder, nor its contents.
+
+2. **Copy or Move** Next's other files and folders (e.g. `pages`, `styles` etc.) into Astro's `src/` folder as you rebuild your site, following [Astro's project structure](/en/core-concepts/project-structure/).
+
+    Like Next, Astro's `src/pages/` folder is a special folder used for file-based routing. All other folders are optional, and you can organize the contents of your `src/` folder any way you like. Other common folders in Astro projects include `src/layouts/`, `src/components`, `src/styles`, `src/scripts`.
+
+
+### Repurpose config file
```

Comment:
If we aren't providing specific guidance for converting a Next config to an Astro one, I'd just call this section 

```suggestion
### The Astro config file
```